draft-ietf-mls-architecture.md

```diff
@@ -1243,10 +1250,10 @@ to a third party. This is often called "non-repudiation".
 
 Roughly speaking, "deniability" is the opposite of "non-repudiation", i.e., the
 property that it is impossible to prove to a third party that a message was sent
-by a given sender.  MLS does not make any claims with regard to deniability.  It
-may be possible to operate MLS in ways that provide certain deniability
-properties, but defining the specific requirements and resulting notions of
-deniability requires further analysis.
+by a given sender. Deniability is not currently provided by the MLS protocol, but the
+protocol avoids constraints that would make it impossible to add deniability
+properties (via extensions) in the future. Defining the specific requirements
```

Comment:
Recommend striking "in the future". This seems to imply that the potential for deniability is linked to follow-on iterations of MLS and cannot be achieved in its current state (i.e. based on implementation decisions). Note that if kept, "in the future" does not imply only to the current time but to the full lifetime of the standard.